app/components/Channels/Channels.js

```diff
@@ -72,8 +77,8 @@ class Channels extends React.Component {
             changeFilter={changeFilter}
           />
         </Panel.Header>
-        <Panel.Body px={4} pt={2} css={{ 'overflow-y': 'overlay', 'overflow-x': 'hidden' }}>
```

Comment:
Yeah. It actually was moved on the grid itself, not removed. But I see a bug. Need to find a way to fix it, since that padding doesn't seem to help anymore (neither on `Panel` nor on the `Grid`)

Reply:
updated